dev-python/nuitka/nuitka-0.6.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6)
```

Comment:
Did/could you try with 3.7?

Reply:
It does work, but the dependency on scons blocks me here.  `dev-util/scons` is still only python 3.6.

Reply:
when the scons maintainer updated this, we can update nuitka. 
@mgorny, mind if I commit a version of scons with python3.7 ? 
according to https://travis-ci.org/SCons/scons/builds/499102473 it should work.

Reply:
> LGTM. setuptools is correctly inherited in RDEPEND, too.
> 
> EDIT: Changed my mind about scons, that package looks like a special case with special care.

Yes, indeed. Scons has some patches in gentoo which need some special care.